database/mongoDatabase.go

```diff
@@ -29,10 +33,19 @@ func (mongoDB *MongoDB) GetClubs() ([]dao.Club, error) {
 	return clubs, nil
 }
 
-func NewMongoDB(client *mongo.Client) *MongoDB {
-	return &MongoDB{client}
+func (mongoDB *MongoDB) PutRating(rating dao.Rating) error {
+	collection := mongoDB.client.Database("mobile-app").Collection("ratings")
+
+	filter := bson.M{"clubId": rating.ClubID, "userId": rating.UserID}
+	update := bson.M{"$set": bson.M{"value": rating.Value}}
```

Comment:
Make "$set" a const string and use bson properties of Rating struct to remove "value"